pyproject.toml

```diff
@@ -209,7 +209,7 @@ pinned = [
     "uritools==4.0.2",
     "urllib3==2.2.0",
     "wcwidth==0.2.13",
-    "webauthn==1.11.1",
+    "webauthn==2.0.0",
```

Comment:
The library is already available in version 2.1.1. Perhaps we want to upgrade it straight away! :)

Reply:
Makes sense :)